tensorflow_io/bigquery/ops/bigquery_ops.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/core/framework/common_shape_fns.h"
+#include "tensorflow/core/framework/op.h"
+
+namespace tensorflow {
+
+REGISTER_OP("BigQueryClient")
+    .Attr("container: string = ''")
+    .Attr("shared_name: string = ''")
+    .Output("client: resource")
+    .SetShapeFn(shape_inference::ScalarShape);
+
+REGISTER_OP("BigQueryReadSession")
+    .Input("client: resource")
+    .Attr("parent: string")
+    .Attr("project_id: string")
+    .Attr("table_id: string")
+    .Attr("dataset_id: string")
+    .Attr("selected_fields: list(string) >= 1")
+    .Attr("output_types: list(type) >= 1")
+    .Attr("requested_streams: int")
+    .Attr("row_restriction: string = ''")
+    .Attr("container: string = ''")
+    .Attr("shared_name: string = ''")
+    .Output("streams: string")
+    .Output("avro_schema: string")
+    .SetShapeFn([](tensorflow::shape_inference::InferenceContext* c) {
+      c->set_output(0, c->Vector(1));
```

Comment:
I think It should be Unknown (c->Vector(c->UnknownDim())).

Reply:
updated and confirmed that it works as expected